dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -28,42 +28,42 @@ MENU_002 MENUEX
 BEGIN
     POPUP ""
     BEGIN
-        POPUP "&Afișare", FCIDM_SHVIEW_VIEW
+        POPUP "&Vizualizare", FCIDM_SHVIEW_VIEW
         BEGIN
-            MENUITEM "D&ale", FCIDM_SHVIEW_BIGICON
-            MENUITEM "&Pictograme", FCIDM_SHVIEW_SMALLICON
-            MENUITEM "&Listă", FCIDM_SHVIEW_LISTVIEW
-            MENUITEM "&Detailii", FCIDM_SHVIEW_REPORTVIEW
+            MENUITEM "Dia&film", FCIDM_SHVIEW_BIGICON
+            MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
+            MENUITEM "&Listare", FCIDM_SHVIEW_LISTVIEW
+            MENUITEM "&Detalii", FCIDM_SHVIEW_REPORTVIEW
         END
         MENUITEM "", -1, MFT_SEPARATOR
-        POPUP "&Ordonare după", FCIDM_SHVIEW_ARRANGE
+        POPUP "Aranjare &pictograme după", FCIDM_SHVIEW_ARRANGE
         BEGIN
             MENUITEM "", -1, MFT_SEPARATOR
-            MENUITEM "Criteriu &automat", FCIDM_SHVIEW_AUTOARRANGE
-            MENUITEM "Aliniază la &grilă", FCIDM_SHVIEW_ALIGNTOGRID
+            MENUITEM "&Aranjare automată", FCIDM_SHVIEW_AUTOARRANGE
+            MENUITEM "Aliniere la &grilă", FCIDM_SHVIEW_ALIGNTOGRID
         END
-        MENUITEM "Împ&rospătează", FCIDM_SHVIEW_REFRESH
+        MENUITEM "&Reîmprospătare", FCIDM_SHVIEW_REFRESH
         MENUITEM "", -1, MFT_SEPARATOR
-        MENUITEM "&Lipește", FCIDM_SHVIEW_INSERT
-        MENUITEM "Lipește ca sc&urtătură", FCIDM_SHVIEW_INSERTLINK
+        MENUITEM "&Lipire", FCIDM_SHVIEW_INSERT
+        MENUITEM "Lipire c&omandă rapidă", FCIDM_SHVIEW_INSERTLINK
     END
 END
 
 /* menubar EDIT menu */
 MENU_003 MENU
 BEGIN
-    MENUITEM "&Revocare\tCtrl+Z", FCIDM_SHVIEW_UNDO
+    MENUITEM "&Anulare\tCtrl+Z", FCIDM_SHVIEW_UNDO
     MENUITEM SEPARATOR
-    MENUITEM "&Decupează\tCtrl+X", FCIDM_SHVIEW_CUT
-    MENUITEM "&Copie\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "&Lipește\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Lipește sc&urtătură", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "&Decupare\tCtrl+X", FCIDM_SHVIEW_CUT
+    MENUITEM "&Copiere\tCtrl+C", FCIDM_SHVIEW_COPY
+    MENUITEM "&Lipire\tCtrl+V", FCIDM_SHVIEW_INSERT
+    MENUITEM "Lipire c&omandă rapidă", FCIDM_SHVIEW_INSERTLINK
     MENUITEM SEPARATOR
-    MENUITEM "Copi&ere în folderul…", FCIDM_SHVIEW_COPYTO
-    MENUITEM "&Mutare în folderul…", FCIDM_SHVIEW_MOVETO
+    MENUITEM "Copiere în &folderul…", FCIDM_SHVIEW_COPYTO
+    MENUITEM "M&utare în folderul…", FCIDM_SHVIEW_MOVETO
     MENUITEM SEPARATOR
-    MENUITEM "Selectează t&ot\tCtrl+A", FCIDM_SHVIEW_SELECTALL
-    MENUITEM "Inversea&ză selecția", FCIDM_SHVIEW_INVERTSELECTION
+    MENUITEM "Selectare &totală\tCtrl+A", FCIDM_SHVIEW_SELECTALL
+    MENUITEM "In&versare selecţie", FCIDM_SHVIEW_INVERTSELECTION
```

Comment:
Did you take all of those accel changes from MS Windows 2k3sp2 romanian? Some of them seem ok to me, but at least FCIDM_SHVIEW_CUT &D would be different to english (which uses &t for that), and &t for FCIDM_SHVIEW_SELECTALL would be different too compared to en-US. **Can you post a screenshot please of that menubar from MS 2k3sp2 romanian for confirmation of that difference.**

Same question for
MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
That also seems to be different compared to en-US where the &M is used for MoveToFolder.
Also a screenshot for that please!

I really don't want to nag you, but this is really a can of worms with those menu accelerators. We must be careful to follow MS to the point for all languages or we will have to touch that many many times again.

Reply:
![2024-03-26 23_44_30-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/f26cd7f7-511f-4aec-b88a-45de995d4b76)

Reply:
MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
and 
"&Decupare" for IDS_CUT confirmed. Thx! Resolved.